src/IssueViz.Security/OpenInIDE_Hotspots/HotspotsList/ViewModels/OpenInIDEHotspotsControlViewModel.cs

```diff
@@ -100,7 +115,7 @@ public IOpenInIDEHotspotViewModel SelectedHotspot
         /// </summary>
         private void AllowMultiThreadedAccessToHotspotsList()
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
+            threadHandling.ThrowIfNotOnUIThread();
```

Comment:
I debated whether just to remove the `ThreadHelper.ThrowIfNotOnUIThread();` call, but decided to leave it in the end. It might catch problems in the future if the callling code changes.